_rules/heading-descriptive-b49b2e.md

```diff
@@ -27,21 +27,16 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [semantic][semantic role] `heading` element that is either [visible][] or [included in the accessibility tree][].
+This rule applies to any [semantic][semantic role] `heading` element that is [included in the accessibility tree][].
 
-## Expectation 1
+## Expectation
 
-For each target element at least one of the following is true:
+Each target element describes the topic or purpose of the first node after the test target for which all the following are true:
+- The node is not [decorative][]; and
+- The node is [palpable content][] (such as text nodes or elements like `h1`, `p`, `img`. etc.); and
+- The node is [included in the accessibility tree].
 
-- the target element is not [visible][]; or
-- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after the target element in tree order in the [flat tree][].
-
-## Expectation 2
-
-For each target element at least one of the following is true:
-
-- the target element is not [included in the accessibility tree][]; or
-- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [included in the accessibility tree][], and after the target element in tree order in the [flat tree][].
+The order of elements is determined by the [flat tree][].
```

Comment:
I'd rather keep the list short and focused and move the precision below it:
```
Each target element describes the topic or purpose of the first node after the test target for which all the following are true:
- The node is not [decorative][]; and
- The node is [palpable content][]; and
- The node is [included in the accessibility tree].

The order of elements is determined by the [flat tree][].
Palpable content is text nodes or elements like `h1`, `p`, `img`, etc.
```
the reason being that we already have a link to the actual definition in the list item, and the precision then kind of breaks the flow of reading.
Not a strong feeling on that; I'm a technical person and have no problem reading further HTML specs…

Reply:
@colabottles and I talked about this, we prefer the current one. As you don't seem to feel strongly on it, we're leaving this as is.